src/app/reporting/Engine.cpp

```diff
@@ -643,7 +643,13 @@ bool Engine::MergeOverlappedAttributePath(AttributePathParams & aAttributePath)
         }
         if (aAttributePath.IsAttributePathSupersetOf(*path))
         {
+            // TODO: the wildcard input path may be superset of next paths in globalDirtySet, it is fine at this moment, since
+            // when building report, it would use the first path of globalDirySet to compare against interested paths read clients
```

Comment:
```suggestion
            // when building report, it would use the first path of globalDirtySet to compare against interested paths read clients
```